### DISCUSSION TITLE
Add "Backfill new or renamed columns" option to Airbyte API

### DISCUSSION THREAD

**Discussion by jackonawalk** [airbytehq/airbyte#44500] [Platform Features or Ideas]
### Description

The connection option "Backfill new or renamed columns" isn't available nor documented for the Airbyte API. Could you advise if this is a hidden option or if it isn't currently support via the API?

https://reference.airbyte.com/reference/createconnection

![Screenshot 2024-08-20 at 4 37 19 PM](https://github.com/user-attachments/assets/ac854663-b292-4ef6-9b5c-bfc2f99a1568)

**Comment by nataliekwong** [comment]
Hi there, this is not currently supported via the API. I will convert this into a discussion as this would be a feature request.

[a comment by an automated account is not shown]

**Comment by sherifkayad** [comment]
I do believe that's still relevant
